discord_bots/main.py

```diff
@@ -280,6 +281,11 @@ async def setup():
     if config.ECONOMY_ENABLED:
         prediction_task.start()
     sigma_decay_task.start()
+    trueskill_setup(
```

Comment:
tau is stored in the env and not per player, so calling setup globally affects the later calculation